CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+<!--
+SPDX-FileCopyrightText: © 2020 Liferay Inc. <https://liferay.com>
+
+SPDX-License-Identifier: MIT
+-->
+
```

Comment:
Can't put this in generated file, it will get blown away.

(In any case, why would we want to claim copyright over a generated file?)

Reply:
Changed in be82591.

Whether or not generated files should have headers, I'll defer to @silverhook.